### PR TITLE
[ Doc ] Update Readme.md

### PR DESCRIPTION
**Pull request by jijoongmoon**
Add new doc which explains NNTrainer internal

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

**Review by jihochu — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -91,9 +96,13 @@ This component defines layers which consist of a neural network model. Layers ha
  | dropout | DropOutLayer | Dropout Layer |
  | embedding | EmbeddingLayer | Embedding Layer |
  | rnn | RNNLayer | Recurrent Layer |
+ | rnncell | RNNCellLayer | Recurrent Cell Layer |
  | gru | GRULayer | Gated Recurrent Unit Layer |
+ | grucell | GRUCellLayer | Gated Recurrent Unit Cell Layer |
  | lstm | LSTMLayer | Long Short-Term Memory Layer |
  | lstmcell | LSTMCellLayer | Long Short-Term Memory Cell Layer |
+ | zoneoutlstm | ZoneoutLSTMLayer | Zoneout Long Short-Term Memory Layer |
```
Comment (lhs8928) [inline README.md:104]: We only support zoneoutlstmcell

**Review by lhs8928 — APPROVED** [review]
LGTM except minor comment

File: README.md (hunk 1 above)
Comment (jijoongmoon) [reply README.md]: fixed.

**Pull request merged**